api/lib/redisHelper.mjs

```diff
@@ -113,5 +113,31 @@ export async function getTotalPossibleScore() {
  * @returns {object} the maximal scores for all assignments so far.
  */
 export async function getMaxScores() {
-    return await getStudentScores('MAX POINTS')
+    return await getStudentScores('MAX POINTS');
 }
+
+/**
+ * Gets a list of all of the students in the class.
+ * Each student is represented as a list: [legalName, email]
+ * @returns {Promise<Array<Array<string>>>} List of [legalName, email]
+ */
+export async function getStudents() {
+    const client = getClient();
+    await client.connect();
+
+    var keys = await client.keys('*@*');
```

Comment:
we can't use getEntry because getEntry requires we know the emails, and the point of this function is to get all of the students data.

Reply:
I believe the difference lies in KEYS vs GET
https://redis.io/docs/latest/commands/keys/
https://redis.io/docs/latest/commands/get/

client.keys(param) returns all keys that match the pattern in param. Since our database has the keys as emails, we need to call the keys method to get all the emails.

client.get(key) returns the value associated with key. This is why in getStudents, I am calling getEntry on each key in the list of keys returned by client.keys("*@*") because we can retrieve their legalName from the key.